src/index.d.ts

```diff
@@ -98,19 +104,29 @@ type ConnectData = {
 }
 
 type TransactionsOptions = {
-  institutionId: string;
+  institutions?: Array<string>;
   from?: string;
   to?: string;
   investments?: string;
   last?: string;
   limit?: number;
+  institutionId?: string;
```

Comment:
Should we remove this now that we already have the `institutions` option that matches the API?

Reply:
Oh I see that it's for backward compat, OK then

Reply:
Changing to institutionIds to be consistent with API